openapi/openapi.yaml

```
@@ -49,381 +57,628 @@ components:
+    description: >-
+      Use email messages to email customers directly.
+  - name: Email notifications
+    description: |-
+      Use email notifications to keep customers
+      informed on events, and to inform them of events that may
+      require action on their end. Email notifications are targeted
+      email messages that can be plain text or HTML.
+
+      Email notifications can also be used to notify teammates about
+      new customers, blocklist matches, risk score changes, and more.
+      For more information, see
+      [Email notifications](https://www.rebilly.com/docs/automations/email-notifications/).
   - name: Plaid credentials
-    description: |
-      Plaid account credentials. Use the Plaid integration to verify a customer's bank credentials, and simplify the process of providing credentials for your customers. For more information, see [Plaid integration](enables merchants to verify a customer's bank credentials, and simplifies the process of providing these credentials for the customer).
+    description: |-
+      Use these operations to manage your Plaid account credentials.
+      The Plaid integration provides customer bank credential
+      verification, and simplifies the process of providing
+      credentials for your customers. For more information, see
+      [Plaid integration](https://www.rebilly.com/docs/automations/plaid-integration/).
   - name: Experian credentials
-    description: |
-      Experian account credentials. Use the Experian integration for identity verification. For more information, see [Experian integration](https://www.rebilly.com/docs/risk/tutorial/experian-integration/#experian-integration).
+    description: |-
+      Use these operations to manage your Experian account
+      credentials. The Experian integration provides identity
+      verification. For more information, see
+      [Experian integration](https://www.rebilly.com/docs/automations/experian-integration/#experian-integration).
   - name: TaxJar credentials
-    description: |
-      TaxJar account credentials. Use the TaxJar integration to calculate taxes and create transactions for the Sales Tax AutoFile feature in your TaxJar account. For more information, see [TaxJar integration](https://www.rebilly.com/docs/billing/concept/taxes/#taxjar-integration).
+    description: |-
+      Use these operations to manage your TaxJar account credentials.
+      The TaxJar integration enables you to calculate taxes and create
+      transactions for the Sales Tax AutoFile feature in your TaxJar
+      account. For more information, see
+      [TaxJar integration](https://www.rebilly.com/docs/automations/tax-jar-integration/#taxjar-integration).
   - name: Metadata
-    description: |
-      Metadata is data that provides structured information on other data.
+    description: >-
+      Use these operations to retrieve and manage metadata. Metadata
+      is data that structured information on other data.
```

```suggestion
      is data that structures information on other data.
```
not sure, but initial sentence sounds inconsistent